lib/developer_portal/app/views/developer_portal/docs.html.liquid

```diff
@@ -1,21 +1,20 @@
-<h1>Documentation</h1>
+{% content_for javascripts %}
+    {{ 'active_docs.js' | javascript_include_tag }}
+{% endcontent_for %}
 
-<p>Use our live documentation to learn about the Echo API</p>
+{% assign spec = provider.api_specs.first %}
```

Comment:
If the provider must have at least one api_spec to reach this view, then this should be:
```suggestion
{% assign spec = provider.api_specs.first! %}
```

Reply:
They don't.